eve/workers/pod-unit-tests/ui/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM centos:7
```

Comment:
Out of scope for this PR, but we may want to open a debt ticket (cc @ChengYanJin) to make sure this worker has run `npm ci` and is only rebuilt when the `package-lock.json` changes. Would reduce the impact of these tests when nothing changed in `ui/`.